ivy_tests/test_ivy/test_frontends/test_torch/test_spectral_ops.py

```diff
@@ -0,0 +1,44 @@
+# global
+from hypothesis import strategies as st
+
+# local
+import ivy_tests.test_ivy.helpers as helpers
+from ivy_tests.test_ivy.helpers import handle_frontend_test
+
+
+# hann_window
+@handle_frontend_test(
+    fn_tree="torch.hann_window",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=["int32", "int64"],
+        max_num_dims=0,
+        min_value=5,
+        max_value=10,
+    ),
+    periodic=st.booleans(),
+    # dtype=helpers.get_dtypes("float", full=False),

```

Comment:
again, this seems to be the same error as reviewed in a different PR (https://github.com/unifyai/ivy/pull/21731#discussion_r1318162968)